packages/suite/src/views/wallet/nfts/NftsTable/NftsRow.tsx

```diff
@@ -0,0 +1,227 @@
+
+import { NftType } from 'src/utils/wallet/nftUtils';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { SUITE } from 'src/actions/suite/constants';
+
+import { useTranslation } from 'src/hooks/suite';
+import styled from 'styled-components';
+import { spacingsPx } from '@trezor/theme';
+import { selectIsCopyAddressModalShown } from 'src/reducers/suite/suiteReducer';
+import { openModal } from 'src/actions/suite/modalActions';
+import { copyToClipboard } from '@trezor/dom-utils';
+import { notificationsActions } from '@suite-common/toast-notifications';
+import { AddressType } from '@suite-common/wallet-types';
+
+type NftsRowProps = {
+    nft: Token;
+    type: NftType;
+    network: Network;
+    shown?: boolean;
+};
+
+const ContractAddress = styled.div`
+    display: inline-block;
+    max-width: 200px;
+    word-break: break-all;
+    white-space: wrap;
+`;
+
+const IconWrapper = styled.div`
+    display: inline-block;
+    margin-left: ${spacingsPx.xxs};
+`;
+
+const NftsRow = ({ nft, type, network, shown }: NftsRowProps) => {
+    const dispatch = useDispatch();
+
+    const { translationString } = useTranslation();
+    const shouldShowCopyAddressModal = useSelector(selectIsCopyAddressModalShown);
+
+    const getNftContractExplorerUrl = (network: Network, nft: Token) => {
+        const explorerUrl = network.explorer.account;
+        const contractAddress = nft.contract;
+        const queryString = network.explorer.queryString ?? '';
+
+        return `${explorerUrl}${contractAddress}${queryString}`;
+    };
+
+    const getNftExplorerUrl = (network: Network, nft: Token, id?: string) => {
+        const explorerUrl = network.explorer.nft;
+        const contractAddressWithId = nft.contract + `/${id}`;
+
+        return `${explorerUrl}${contractAddressWithId}`;
+    };
+
+    const onCopyAddress = (address: string, addressType: AddressType) => {
+        if (shouldShowCopyAddressModal) {
+            dispatch(
+                openModal({
+                    type: 'copy-address',
+                    addressType,
+                    address,
+                }),
+            );
+        } else {
+            const result = copyToClipboard(address);
+            if (typeof result !== 'string') {
+                dispatch(notificationsActions.addToast({ type: 'copy-to-clipboard' }));
+            }
+        }
+    };
+
+    const name = nft.name.length > 15 ? `${nft.name.slice(0, 15)}...` : nft.name;
+
+    return (
+        <Table.Row>
+            <Table.Cell colSpan={1}>
+                <TruncateWithTooltip>{name}</TruncateWithTooltip>
+            </Table.Cell>
+            {type === 'ERC1155' && (
+                <>
+                    <Table.Cell colSpan={1}>
+                        <Column alignItems="flex-start">
+                            {nft.multiTokenValues?.map(value => (
+                                <Row key={value.id?.toString()}>
+                                    <Link
```

Comment:
TODO: Please check if we can use `<Link...` in Suite, it ignores tor I think. I guess `<TrezorLink...` should be used everywhere